src/common/translation/translation-set.ts

```diff
@@ -260,6 +260,8 @@ export interface TranslationSet {
     windowsShutdownDescription: string;
     windowsRestart: string;
     windowsRestartDescription: string;
+    windowsReboot: string;
+    windowsRebootDescription: string;
```

Comment:
I think the `windowsRebootDescription` is not used anywhere so I suggest that we remove it.

Reply:
Yeah, you're right. I guess I just did it that way for it to look more consistent, but yes, it isn't useful at all. Will make the changes right away